kahuna/public/js/crop/controller.js

```diff
@@ -37,22 +37,23 @@ crop.controller('ImageCropCtrl', [
     keyboardShortcut,
     defaultCrop,
     cropOptions,
```

Comment:
It looks like we're getting crop options from `cropSettings.getCropOptions()` now. Can we remove the `cropOptions` import now?